plugin/ScriptLoader/ScriptLoaderGenericConfigurationDefault.php

```diff
@@ -60,13 +66,199 @@ public function getScriptLoaderGenericConfiguration(): array
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true);
+		$scripts[] = (new IncludeFileModel($script, 'onoffice-prevent-double-form-submission', plugins_url('/dist/onoffice-prevent-double-form-submission.min.js', $pluginPath)))
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true);
+		$scripts[] = (new IncludeFileModel($script, 'onoffice-form-preview', plugins_url('/dist/onoffice-form-preview.min.js', $pluginPath)))
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true);
+
+		if (get_option('onoffice-settings-honeypot') == true) {
+			$scripts[] = (new IncludeFileModel($script, 'onoffice-honeypot', plugins_url('/dist/onoffice-honeypot.min.js', $pluginPath)))
+					->setDependencies(['jquery'])
+					->setLoadInFooter(true);
+		}
+
+		$this->localizeFormPreviewScript();
+
+		return $scripts;
+	}
+
+	/**
+	 * @param array $scripts
+	 * @param string $pluginPath
+	 * @param string $script
+	 * @param string $style
+	 * @param string $defer
+	 * @return array
+	 */
+	private function renderScriptFormPageForEstateDetailPage(array $scripts, string $pluginPath, string $script, string $style, string $defer): array
+	{
+		$scripts[] = (new IncludeFileModel($script, 'slick', plugins_url('/third_party/slick/slick.js', $pluginPath)))
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true)
+				->setLoadAsynchronous($defer);
+		$scripts[] = (new IncludeFileModel($script, 'onoffice_defaultview', plugins_url('/dist/onoffice_defaultview.min.js', $pluginPath)))
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true);
+		$scripts[] = new IncludeFileModel($style, 'slick', plugins_url('/third_party/slick/slick.css', $pluginPath));
+		$scripts[] = new IncludeFileModel($style, 'slick-theme', plugins_url('/third_party/slick/slick-theme.css', $pluginPath));
+
+		return $scripts;
+	}
+
+	/**
+	 * @param array $scripts
+	 * @param string $pluginPath
+	 * @param string $script
+	 * @param string $style
+	 * @param string $defer
+	 * @return array
+	 */
+
+	private function renderScriptForEstateListPage(array $scripts, string $pluginPath, string $script): array
+	{
+		$scripts[] = (new IncludeFileModel($script, 'onoffice-sort-list-selector', plugins_url('/dist/onoffice-sort-list-selector.min.js', $pluginPath)))
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true);
+		$scripts[] = (new IncludeFileModel($script, 'onoffice-form-preview', plugins_url('/dist/onoffice-form-preview.min.js', $pluginPath)))
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true);
+		$scripts[] = (new IncludeFileModel($script, 'onoffice-custom-select', plugins_url('/dist/onoffice-custom-select.min.js', $pluginPath)))
+				->setDependencies(['jquery'])
+				->setLoadInFooter(true);
+		$scripts[] = (new IncludeFileModel($script, 'onoffice-multiselect', plugins_url('/dist/onoffice-multiselect.min.js', $pluginPath)))
```

Comment:
multiselect and estatetype were loaded asyncron before. I think setLoadAsynchronous is missing, or?